src/api/user.ts

```diff
@@ -29,6 +30,27 @@ const userAPI = {
       LOGGER.Error(e as string);
       return res.status(500).json(response(requestTime, 'Internal server error', null, e))
     }
+  },
+  login: async (req: Request, res: Response) : Promise<Response | undefined> => {
+      const requestTime = moment().tz(CONSTANT.WIB).format(CONSTANT.dateFormat);
+      const errors = validationResult(req);
+      if (!errors.isEmpty()) return res.status(400).json(response(requestTime, 'Value in body missing the validation requirement', null, errors.array()));
+
+      const loggedUser: LoginUser = req.body;
+      try {
+          const user = await UserModel.findOne(loggedUser.email);
+          if (!user) return res.status(404).json(response(requestTime, 'User with given email does not exists'));
```

Comment:
security reason tips: jangan dikasih tau errornya apa lebih baik 'Email or password is incorrect'